src/teleop/gui/src/components/ControlGUI.vue

```diff
@@ -1,13 +1,66 @@
 <template>
-  <div>
+  <div class="wrapper">
+    <div class="box header">
+      <img
+        src="/static/mrover.png"
+        alt="MRover"
+        title="MRover"
+        width="48"
+        height="48"
+      />
+      <h1>ESW Debug</h1>
+      <div class="spacer"></div>
+      <div class="spacer"></div>
+    </div>
     <DriveControls></DriveControls>
```

Comment:
Can we move these and the gimbal controls to bottom, it's a bit weird having these components with no visible html in the middle of all the grid divs.